packages/gatsby-plugin-google-tagmanager/src/gatsby-node.js

```diff
@@ -40,4 +40,5 @@ exports.pluginOptionsSchema = ({ Joi }) =>
       .description(
         `Name of the event that is triggered on every Gatsby route change.`
       ),
+    disableWebVitalsTracking: Joi.boolean().default(true),
```

Comment:
We will make it a default in the next major. I want to do some more updates in a follow up PR